web/modules/custom/hoeringsportal_config_settings/src/Form/ItkGeneralSettingsForm.php

```diff
@@ -173,6 +173,27 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('users_manual_url'),
     ];
 
+    $node_newsletter = Node::load($config->get('node_newsletter') ?? -1);
+    $form['misc']['node_newsletter'] = [
```

Comment:
`newsletter_node` may be a better name (and give some sort of namespacing matching `newsletter_iframe_source` and `newsletter_iframe_height`).

A description telling that this is the page on which to show the newsletter sign-up (if the code reviewer understands the code correctly) will be useful.

Reply:
Added description